File: src/main/java/com/jicay/bookmanagement/domain/usecase/BookUseCase.java
package com.jicay.bookmanagement.domain.usecase;

import com.jicay.bookmanagement.domain.model.Book;

import java.util.List;

public class BookUseCase {
    public List<Book> getAllBooks() {
        return List.of(
                new Book("Les Misérables", "Victor Hugo"),
                new Book("Hamlet", "William Shakespeare1")
        );
    }
}

File: src/main/kotlin/com/jicay/bookmanagement/domain/usecase/BookUseCase.kt
package com.jicay.bookmanagement.domain.usecase

import com.jicay.bookmanagement.domain.model.Book

class BookUseCase {
    fun getAllBooks(): List<Book> {
        return listOf(
            Book("Les Misérables", "Victor Hugo"),
            Book("Hamlet", "William Shakespeare")
        ).sortedBy {
            it.name.lowercase()
        }
    }
}